rails_app/app/views/homepage/index.html.haml

```diff
@@ -0,0 +1,8 @@
+.well
+  .row
+    .col-xs-6.col-xs-offset-3
+      .input-group
+        %input.form-control{type: 'text', placeholder: 'www.example.com'}
+        %span.input-group-btn
+          %button.btn.btn-default{type: 'button', type: 'button'}
+            Wyszukaj!
```

Comment:
Dude, we're trying to keep this international, please stick to English

Reply:
Kk